data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
But you can break any server by modifying a script.
The amount of damage might differ but you could crash 1.1+ by not knowing how to handle userdata values.

You can also break 0.x servers via Lua scripts.
The point is to make people learn, we cant make everything noob proof without having a billion checks for everything the server handles.

I agree that this has gotten out of hand and it would be better to talk abt this on otland to see what ppl think.